typescript/postgis-express/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "scripts": {
     "dev": "ts-node src/index.ts",
-    "seed": "ts-node prisma/seed.ts",
+    "seed": "prisma db seed --preview-feature",
```

Comment:
Since example READMEs are like mini-tutorials, let's let the user run Prisma's seeding command

Reply:
Let's also update to the latest version of jest